src/App.js

```diff
@@ -12,11 +12,13 @@ import './App.css';
 class App extends Component {
   state = { isLoggedIn: false, loginBypass: false, bitgo: null };
 
-  updateLoginState = (isLoggedIn) => (bitgoInstance) => {
+  updateLoginState = (isLoggedIn) => (bitgoInstance, utxoLibInstance) => {
     if (isLoggedIn && !bitgoInstance) {
       throw new Error('If logging in, please pass in an authenticated BitGoJS instance.');
     }
 
+    bitgoInstance.utxoLib = utxoLibInstance;
```

Comment:
I don't think we need a global utxolib object, as utxolib is stateless. You should be able to just `import ... from 'bitgo-utxo-lib'` wherever you need it.

We only create a global SDK object so that we only have to authenticate once.

Reply:
I would have liked to do that, but we need to use the `hdPath` function for deriving v1 addresses correctly, which gets patched onto bitgo-utxo-lib just before bitgojs re-exports it: https://github.com/BitGo/BitGoJS/blob/master/src/bitcoin.js

I've opened a PR against bitgo-utxo-lib to move these functions out of bitgojs and into there: https://github.com/BitGo/bitgo-utxo-lib/pull/16, but I don't want to block this change on getting that into bitgo-utxo-lib, given that we already have access to an instance at `BitGoJS.bitcoin`.

Reply:
Ah ok, fair enough then. Although if we move createRecoveryTx into BitGoJS (see next comment) we could still probably remove this.